app/config/FrontendAppConfig.scala

```diff
@@ -138,7 +138,7 @@ class FrontendAppConfig @Inject()(val servicesConfig: ServicesConfig, runModeCon
 
   lazy val vatRatesUrl = "https://www.gov.uk/guidance/rates-of-vat-on-different-goods-and-services"
 
-  lazy val invalidAffinityUrl = "https://www.tax.service.gov.uk/business-account/add-tax/vat/registered/no"
+  lazy val invalidAffinityUrl = s"$host/register-for-vat/error/individual-affinity"
```

Comment:
This isn't needed anymore, the line in BaseController that uses it can just be replaced with
          Future.successful(Redirect(routes.IndividualAffinityKickOutController.show()))
Will prolly need to fix a test if you change that.